ontology/docs/systems_and_equipment.md

```diff
@@ -0,0 +1,67 @@
+# Modeling Systems and Equipment
+
+This project aims to provide strong guidance in modeling; this is to ensure that all models retain certain commonalities, even while being constructed by disparate modelers with different system contexts. This document is intended to outline some common system types and how they may be modeled effectively and consistently.
+
+
+## Modeling Domains
```

Comment:
suggest having sections at this level by namespace.  (obviously leaving other domains as a TODO for now)